doc/src/build/json-rpc.md

```diff
@@ -201,11 +201,11 @@ curl --location --request POST $SUI_RPC_HOST \
               "method": "sui_moveCall",
               "params": [
                   "{{owner_address}}",
-                  "0000000000000000000000000000000000000002",
+                  "0x2",
```

Comment:
Do we allow short form everywhere?

Reply:
yes, I have done some test manually and a unit test here https://github.com/MystenLabs/sui/blob/d4d894e7dbb1e9a0d763649a3bf1d85fc55d4710/sui_types/src/unit_tests/base_types_tests.rs#L198-L215

Should be allowed everywhere